src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+@import "./card.scss";
+@import "./stars.scss";
+@import "../utils/variables";
+
+* {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
```

Comment:
it is redundant to reset margin and padding for all selectors. reset only elements which really need it

Reply:
Still not fixed